### DISCUSSION TITLE
Mac Voice Over - Not Working With SEB Client

### DISCUSSION THREAD

**Discussion by sam-lee-ku** [SafeExamBrowser/seb-mac#431] [Q&A]
Hello, my team works on a testing platform for students. One of the complaints we received was Mac Voice Over not being usable with our client (which we use SEB for). I also tried the issue myself:

When I try to use Mac Voice Over with SEB client, I am not able to run certain commands. These commands include Caps Lock (modifier) + L, Caps Lock (modifier) + A, Caps Lock (modifier) + right arrow, Caps Lock (modifier) + Space.

I am wondering if the SEB client (and or browser) blocks any keys from being used when you run it? Or could there be other reasons why Mac Voice Over doesn't work with SEB client?

I also want to add that I've used Mac Voice Over in Safari and Chrome, and I was able to navigate and select answers as needed, but not in SEB.

**Comment by danschlet** [comment]
We tested VoiceOver only with the standard VO modifier Control and Option keys pressed at the same time, not with Caps Lock.

But SEB does not modify VoiceOver behavior. It is possible though that the lockdown mode of SEB prevents some VO commands to work. 

You could try:
- Change the VO modifier to the Control and Option keys pressed at the same time and test if these commands work then.
- Use the [AAC Assessment Mode](https://safeexambrowser.org/macosx/mac_usermanual_en.html#SecurityPane) instead of the SEB custom lockdown mode. Maybe this macOS native lockdown mode has better compatibility with VO.
- Please report your results here. We're happy to improve something in SEB if it is possible. But we have limited resources and cannot investigate this issue right now.
- It would be helpful if you would list what those VO commands are supposed to do. I don't have a full overview of all VO commands and [Apple's documentation](https://support.apple.com/guide/voiceover/about-voiceover-commands-and-gestures-vo14111/10/mac/14.0) is cumbersome to find specific commands.
